src/webkitgtk/mod.rs

```diff
@@ -600,6 +600,68 @@ impl InnerWebView {
     Ok(())
   }
 
+  pub fn bounds(&self) -> Rect {
+    let mut bounds = Rect {
+      x: 0,
+      y: 0,
+      width: 0,
+      height: 0,
+    };
+
+    if let (Some(xlib), Some(display), Some(window_handle)) =
+      (&self.xlib, self.x11_display, self.x11_window)
+    {
```

Comment:
We can probably skip using x11 and use `self.gtk_window` directly if it is a child

Reply:
I tried that, but I use i3wm and by default it expands the window to fill the tile, so the bounds wrong without using x11. if we don't really care about tiling window managers then yeah we can skip to gtk directly

Reply:
it is fine to keep then